dev-packages/node-integration-tests/suites/child-process/fork.mjs

```diff
@@ -12,6 +12,7 @@ Sentry.init({
   transport: loggingTransport,
 });
 
+// eslint-disable-next-line no-unused-vars
 const _child = fork(path.join(__dirname, 'child.mjs'));
```

Comment:
l: do we need the disable comment here? or could we just get rid of the variable?
(feel free to ignore, as it's not the main focus of this PR)

```suggestion
fork(path.join(__dirname, 'child.mjs'));
```

Reply:
yeah, I first did not want to touch this because 🤷 but let's clean this up now, seems to be working just fine without this!